endpointmanager/pkg/endpointmanager/postgresql/fhirendpointinfostore_test.go

```diff
@@ -219,6 +331,22 @@ func Test_PersistFHIREndpointInfo(t *testing.T) {
 		t.Errorf("did not expected endpoint in db")
 	}
 
+	err = store.DeleteFHIREndpointInfo(ctx, endpointInfo1RequestedVersion)
+	if err != nil {
+		t.Errorf("Error deleting fhir endpointInfo: %s", err.Error())
+	}
+
+	_, err = store.GetFHIREndpointInfo(ctx, endpointInfo1RequestedVersion.ID) // ensure we deleted the entry
+	if err == nil {
+		t.Errorf("did not expected endpoint in db")
+	}
+
+	// Need to do this now to pass tests, when requested version entries no longer have metadata entries, remove this delete statement
+	_, err = store.DB.Exec("DELETE FROM fhir_endpoints_metadata WHERE id=$1;", metadataIDRV)
```

Comment:
This comment is a little confusing, does this mean we don't have to do this DELETE statement anymore?

Reply:
This comment may need to be deleted. I think initially we were saying that only the default version would have a metadata entry, but it looks like instead each version entry, including the default version, have their own metadata entries. Therefore, this DELETE statement is still necessary, but the comment is no longer true, so I will remove the comment!

Reply:
sounds good! totally forgot that at one point we weren't going to have a metadata entry for each version...